test/setup/outputs.tf

```diff
@@ -14,15 +14,19 @@
  * limitations under the License.
  */
 
+output "kms_project_id" {
+  value = module.project_ci_kms.project_id
+}
+
 output "project_id" {
   value = module.project_ci_kms.project_id
 }
```

Comment:
These refer to the same variable, can be merged into a single output var

Reply:
+1

`test/setup` shouldn't have configurations that are used specifically for a single example. I recommend to keep just the `project_id` here.

If your example really needs to have an input var called `kms_project_id` instead of `project_id`, consider using [fixtures](https://pkg.go.dev/github.com/GoogleCloudPlatform/cloud-foundation-toolkit/infra/blueprint-test#readme-5-1-1-terraform-fixtures) instead.

Reply:
The reason for two different name is, 
1. For the other resources like cryptoKey, project-id means the kms project where these resources will be configured. And because of that project-id variable make sense.
2. for autokey we deal with two different kind of project 
    1. kms project  and 2. resource project
    To clearly disambiguate between two i have introduced kms_project_id as variable.

I will try use fixtures.

Reply:
done